service/history/transferQueueStandbyProcessor.go

```diff
@@ -166,9 +166,17 @@ func (t *transferQueueStandbyProcessorImpl) process(qTask queueTaskInfo, shouldP
 			err = t.processRecordWorkflowStarted(task)
 		}
 		return metrics.TransferStandbyTaskRecordWorkflowStartedScope, err
+
```

Comment:
plz add test for TransferTaskTypeUpsertWorkflowSearchAttributes for standby processor